deployer/pom.xml

```diff
@@ -54,6 +54,10 @@
             <artifactId>brooklyn-software-webapp</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-camp</artifactId>
```

Comment:
`SeaCloudsMonitoringInitializationPolicies` requires to find the application children which are pointed by the children Id configkey [TARGET_ENTITIES](https://github.com/SeaCloudsEU/SeaCloudsPlatform/pull/300/files#diff-64d88bf2a01ddc5c605f6710156bfdbaR65). Ids can be added to the entities using two different configkeys either [TOSCA_PLAN_ID](https://github.com/SeaCloudsEU/SeaCloudsPlatform/pull/300/files#diff-64d88bf2a01ddc5c605f6710156bfdbaR61) or [CAMP_PLAN_ID](https://github.com/SeaCloudsEU/SeaCloudsPlatform/pull/300/files#diff-64d88bf2a01ddc5c605f6710156bfdbaR108) so I added CAMP dependency to use this constant but as you mentioned this includes more dependencies, and I could remove it.